uaclient/cli.py

```diff
@@ -197,12 +197,17 @@ def _perform_enable(entitlement_name: str, cfg: config.UAConfig, *,
     return ret
 
 
-@assert_attached_root
 def action_enable(args, cfg):
     """Perform the enable action on a named entitlement.
 
     @return: 0 on success, 1 otherwise
     """
+    if os.getuid() != 0:
+        raise exceptions.NonRootUserError()
+    if not cfg.is_attached:
+        raise exceptions.UserFacingError(
```

Comment:
As I said in my bus review, I'd like us to not duplicate this.  Could we refactor this to a helper that `assert_attach_root` could use too? We can just pass an exception instance in, instead of the message (I couldn't see the definition of `assert_attached_root` when I wrote that).